### ISSUE TITLE
🐞 bug: bick default selects darwin when using manjaro

### ISSUE THREAD

**Issue by gaerfield** [zdharma-continuum/zinit#160]
### Issue description

When "bpick" is not given explicitly the automatic selection of a release-binary selects darwin instead of linux.
```
❯ uname -a
Linux ithaka 5.13.19-2-MANJARO #1 SMP PREEMPT Sun Sep 19 21:31:53 UTC 2021 x86_64 GNU/Linux
```
For example [k9s](https://github.com/derailed/k9s/releases) is configured:
```
zcommand from"gh-r"; zload derailed/k9s
```

Installing / zinit update reports:
```
Downloading derailed/k9s…
(Requesting `k9s_Darwin_x86_64.tar.gz'…)
######################################################################## 100,0% 
ziextract: Unpacking the files from: `k9s_Darwin_x86_64.tar.gz'…
ziextract: Successfully extracted and assigned +x chmod to the file: `k9s'.
```

### zinit config

to much to be reported

### zinit version or commit ID

4da69625bcb7c12f929db84af2352e046eb11d75

### zsh version

5.8

### host info

XDG_SESSION_TYPE=x11 OSTYPE=linux-gnu CPUTYPE=x86_64 MACHTYPE=x86_64 5.13.19-2-MANJARO ID=manjaro

**Comment by gaerfield** [comment]
Is it possible to set a fixed zinit version (to "tags/v3.17") to temporarily avoid zinit self-update updating to the latest unstable?

**Comment by vladdoster** [comment]
See the zinit repository README.md for that option.

**Comment by yutkat** [comment]
Now bpick on Linux is broken.
Please revert this commit https://github.com/zdharma-continuum/zinit/commit/4da69625bcb7c12f929db84af2352e046eb11d75

**Comment by gaerfield** [comment]
@vladdoster Thanks for the hint. I was trying to find it before reporting and again now. Still I'm not able to identify the piece of information howto lock zinit to a specific version. I know that this is possible with "ver" for plugins, but for zinit itself I don't see this option.

**Comment by gaerfield** [comment]
For the moment I have edited the function `.zinit-update-or-status-all` in `zinit-autoload.zsh` and commented line that executes a self-update when `zinit update` or `zinit update -p` is issued:
```
# .zinit-self-update -q
```
That prevents at least that zinit is updating itself and afterwards is overwriting my binaries with the darwin-variants during update.

**Comment by AndrewDaws** [comment]
> Now bpick on Linux is broken. Please revert this commit [4da6962](https://github.com/zdharma-continuum/zinit/commit/4da69625bcb7c12f929db84af2352e046eb11d75)

Thanks for the lead there @yutkat as to why my setup was broken as well. I commented and pointed towards this issue in the pull request #158.

Should we make a separate pull request to revert the broken commits all-together, or see if @vladdoster fixes the bpick change?

**Comment by gaerfield** [comment]
@AndrewDaws I consider this bug fixed because the merge of #161
I've tested it in my environment and weren't able to reproduce it (at least "on my machine")
@vladdoster Still I'm looking forward to your pull request. The introduction of tests should stabilize this part.
